packages/protocol/docs/how_taiko_proves_blocks.md

```diff
@@ -278,16 +275,16 @@ h_timestamp(timestamp)
 h_mix_hash(mixHash)
 h_beneficiary(beneficiary)
 h_parent_hash(parentHash)
-h_ommers_hash(ommersHash)
+h_ommers_hash("ommersHash = keccek([])")
 h_state_root(stateRoot)
 h_transactions_root(transactionsRoot)
 h_receipts_root(receiptsRoot)
-h_logs_bloom(logsBloom)
-h_difficulty(difficulty)
-h_extra_data(extraData)
-h_nonce(nonce)
+h_logs_bloom("logsBloom = []")
```

Comment:
not sure the motivation of make `header.logsBloom` to equal to `[]`?

Reply:
Is it correct to say: "`logsBloom` must be a `bytes32[8]` with all zeros"?

Reply:
It seems like the common representation is `bytes[256]` where I look, but yes, 256 `0` bytes.